pages/Styles-Overview.md

```diff
@@ -13,10 +13,13 @@ The `polygons` draw style tessellates and extrudes vector shapes into 3D geometr
 The `lines` draw style can turn either polygonal or line data into lines.
 
 #### `points`
-The `points` draw style draws a square at a point, and can fill the point with either a customizable dot or a `sprite`. It can work with point data, lines, or polygons. Points will "collide" with each other, with only the winner being drawn, determined by the `[priority](draw.md#priority)` draw parameter.
+The `points` draw style draws a square at a point, and can fill the point with either a customizable dot or a `sprite`. It can work with point data, lines, or polygons. Points will "collide" with each other, with only the winner being drawn, determined by the [`priority`](draw.md#priority) draw parameter.
 
 #### `text`
-The `text` draw style draws a rectangle at a point, and paints it with a texture it generates automatically based on its input datasource. It can work with point, line, or polygon data. Text labels will "collide" with each other, with only the winner being drawn, determined by the `[priority](draw.md#priority)` draw parameter.
+The `text` draw style draws a rectangle at a point, and paints it with a texture it generates automatically based on its input datasource. It can work with point, line, or polygon data. Text labels will "collide" with each other, with only the winner being drawn, determined by the [`priority`](draw.md#priority) draw parameter.
```

Comment:
(I realize this text wasn't even part of this PR, but now that I'm reading it...)

I'd prefer (as with the feature selection comments) to not get into implementation details of *how* things are rendered, and stick more to the user-facing syntax and the end result it achieves visually (for instance, this description of how text is rendered isn't strictly how ES does it either). 

Can we simplify to describe that this draws text labels at a given point (for lines, at a point along the line, for polygons, at the centroid)?